virtualenv.py

```diff
@@ -1395,6 +1395,20 @@ def install_python(home_dir, lib_dir, inc_dir, bin_dir, site_packages, clear, sy
                 elif os.path.exists(python_dll_d_dest):
                     logger.info("Removed %s as the source does not exist", python_dll_d_dest)
                     os.unlink(python_dll_d_dest)
+
+            if not IS_PYPY:
```

Comment:
Why just pypy, what about jython? We should also mention why this is so special

Reply:
* Jython is java-based and doesn't provide import libraries like CPython does.
* Pypy indeed provides a `libs` folder with the corresponding `pythonXY.lib` file, I will update the patch to not exclude it.